### DISCUSSION TITLE
화면 공유 후 종료 시 창 위치 및 사이즈

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#46] [제보하기]
1번 (노트복) / 2번 (추가 모니터)

1번 화면에서 메타몽을 실행하던 중
공유 화면을 2번으로 선택하여 공유 한 후에,
Stop을 통해 종료하면
메타몽 창 위치가 2번으로 변경되며 사이즈도 변경이 됩니다. (기존 위치와 기존 사이즈가 아님)

**Comment by ACT-TJ** [comment]
안녕하세요!

공유 수행 전 모니터 위치를 파악해 해당 모니터에 뿌려주는 시도를 1차로 했었는데
잘 되지 않았던 것 같습니다.
개발팀에 공유하고 조치해 볼 수 있도록 하겠습니다~!

감사합니다.
